.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @diesel-rs/reviewers
```

Comment:
Seems we have to give the write access to the team (looking at https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax)?

Reply:
That's unfortunate. We should not need to do that. I will close this PR as this means it won't work that way.